singer-io/tap-xero
# Xero Tap

QA Notes:

- Runtime for demo account sync of all data from 2001: 10.753s
- Total records sent to Stitch: 630
- Total rows replicated by Stitch: 2189

Using the singer-check-tap command, I confirmed that no rows were replicated during the second run except for the streams that sync all rows every time.

- Second run records sent to Stitch: 22
- Total rows replicated by Stitch: 2232

I sanity checked the schemas within Redshift. The only oddity I noticed was that "date" fields had a string datatype in Redshift. Is this expected? "date-time" fields were fine.
